# Request (empty)
---
# Response
bool is_recording
string dataset_path
string[] active_topics
int32[] batch_sizes
int32 pending_batches
float64 disk_usage_gb
string last_error

